src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -101,16 +101,23 @@ public partial struct ObjectEnumerator : System.Collections.Generic.IEnumerable<
             System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         }
     }
+    public abstract partial class JsonException : System.Exception
+    {
+        protected JsonException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
```

Comment:
Why do we need this protected ctor?

Reply:
They are called from the other two exceptions from their private ctor via `base(info, context)`